gps_uart.c

```diff
@@ -204,7 +205,7 @@ GpsUart* gps_uart_enable() {
     gps_uart->baudrate = gps_baudrates[current_gps_baudrate];
     gps_uart->changing_baudrate = false;
     gps_uart->backlight_on = false;
-    gps_uart->speed_units = KNOTS;
+    gps_uart->speed_units = KPH;
```

Comment:
I think it makes more sense to keep the native unit from the NMEA message as the default?

```suggestion
    gps_uart->speed_units = KNOTS;
```